src/components/CurrentRefinements/CurrentRefinements.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'preact-compat';
+import PropTypes from 'prop-types';
+// import cloneDeep from 'lodash/cloneDeep';
+import isEqual from 'lodash/isEqual';
+import upperFirst from 'lodash/upperFirst';
+// import Template from '../Template.js';
+import { isSpecialClick } from '../../lib/utils.js';
+
+function handleClick(cb) {
+  return event => {
+    if (isSpecialClick(event)) {
+      // do not alter the default browser behavior
+      // if one special key is down
+      return;
+    }
+    event.preventDefault();
+    cb();
+  };
+}
+
+class CurrentRefinements extends Component {
+  shouldComponentUpdate(nextProps) {
+    return !isEqual(this.props.refinements, nextProps.refinements);
+  }
+
+  renderItem(refinement, index) {
+    // const attribute = this.props.attributes[refinement.attributeName] || {};
+    // const customTemplateProps = getCustomTemplateProps(attribute);
+    const key =
+      refinement.attributeName +
+      (refinement.operator ? refinement.operator : ':') +
+      (refinement.exclude ? refinement.exclude : '') +
+      refinement.name;
+
+    return (
+      <li className={this.props.cssClasses.item} key={key}>
```

Comment:
Not sure we can make this a template?

Reply:
wait, if a custom template can't remove a filter, it's broken, right?